js/npm/Debug.hx

```diff
@@ -0,0 +1,44 @@
+package js.npm;
+
+import haxe.Log;
+import haxe.PosInfos;
+import js.Lib;
+import js.support.Either;
+
+typedef DebugFunction = Dynamic -> haxe.extern.Rest<Dynamic> -> Void;
+typedef DebugHaxeLogTraceFunction = Dynamic -> ?PosInfos -> Void;
+
+extern class Debug
+implements npm.Package.RequireNamespace<"debug","^2.2.0">
+{
+	public static inline function construct(name : String) : DebugInstance {
+		return new DebugInstance(name);
+	}
+	
+	public static inline function enable(name : String) : Void {
+		untyped Lib.require("debug").enable(name);
```

Comment:
I wish it was that simple. There are many things that should be fulfilled here:

1. Easy instantiation: `var debug = new Debug("myapp")`
2. Easy usage: `debug("Working")`
3. Easy configuration: `debug.log(haxe.Log.trace)`
4. Easy static configuration: `Debug.log(someOtherLog)`

It seems not possible to fulfill all these at the same time. If I use `construct` and an abstract class, 2-4 can be fulfilled, keeping as close to the original API as possible.